keyboards/framework/laptop_16/config.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022-2023 Framework Computer
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Increase to 16K for more storage space
+// With 64K backing size, this spread the wear level over 4 blocks
+   //#define WEAR_LEVELING_LOGICAL_SIZE (4 * 4096)
+// Increase to 64K backing size to allow more wear leveling
+   //#define WEAR_LEVELING_BACKING_SIZE (16 * 4096)
```

Comment:
These comments to increase these sizes seem redundant when these sizes are already configured by default.
```suggestion
```